syntax = "proto3";

package src.shared.types;

message SyncFolder {
    string name = 1;
    string targetFolder = 2;
    string originFolder = 3;
}
